src/App.tsx

```diff
@@ -1,87 +1,60 @@
 import * as React from "react";
-import {useEffect, useState} from "react";
+import { useEffect, useState } from "react";
 import { useFilePicker } from 'use-file-picker';
 
-import {ForceFileData, MarkerFileData} from "./DataTypes";
-import {parseForceFileData, parseMarkerFileData} from "./Parser";
+import { ForceFileData, MarkerFileData } from "./DataTypes";
+import { parseForceFileData, parseMarkerFileData } from "./Parser";
 import RenderView from "./RenderView";
 
 import "./App.scss";
 
 
-const emptyMarkerFileData: MarkerFileData = {
-	markers: [],
-	frames: []
-};
-
-const emptyForceFileData: ForceFileData = {
-	frames: []
-}
-
-
 export default function App() {
 
-	const [openMarkerFileSelector, {plainFiles: [markerFile], loading: markersLoading}] = useFilePicker({
-		accept: ['.txt','.tsv','.csv','.mot']
-	});
-
-	const [openForceFileSelector, {plainFiles: [forceFile], loading: forcesLoading}] = useFilePicker({
-		accept: ['.txt','.tsv','.csv','.mot']
-	});
-
-	// eslint-disable-next-line prefer-const
-	let [markerFileData, setMarkerFileData] = useState<MarkerFileData>();
+	/* Load and parse provided marker file into markerFileData */
+	const [openMarkerFileSelector, {plainFiles: [markerFile], loading: markersLoading}] = useFilePicker({accept: ['.txt','.tsv','.csv']});
+	const [markerFileData = {markers: [], frames: []}, setMarkerFileData] = useState<MarkerFileData>();
```

Comment:
this seems like a confusing way to format this, why not just make the blank data the initial state?

Reply:
I must've missed that line. This was one that I also refactored but I must not have copied over.